code/web/src/modules/user/EditProfile.js

```diff
@@ -0,0 +1,313 @@
+
+  // onSubmit = (event) => {
+  //   event.preventDefault()
+    
+  //   this.setState({
+  //     isLoading: true,
+  //   })
+
+  //   this.props.messageShow('Saving information, please wait...')
+    
+  //   //call to back-end to post/update new data (method below does not exist yet)
+  //   this.props.updateProfileInfo(this.state.newProfileData)
+  //     .then((response) => {
+  //       this.setState({
+  //         isLoading: false,
+  //       })
+        
+  //       if (response.data.errors && response.data.errors.length > 0) {
+  //         this.props.messageShow(response.data.errors[0].message)
+  //       } else {
+  //         this.props.messageShow('Information saved successfully.')
+  //         //might need something else here
+  //         //might need to save image path on user
+  //       }
+  //     })
+
+  //     .catch((error) => {
+  //       this.props.messageShow('There was some error. Please try again.')
+
+  //       this.setState({
+  //         isLoading: false,
+  //       })
+  //     })
+  //     .then(() => {
+  //       window.setTimeout(() => {
+  //         this.props.messageHide()
+  //       }, 5000)
+  //     })
+  // }
+
+  // onUpload = (event) => {
+  //   this.props.messageShow('Uploading photo, please wait...')
+
+  //   this.setState({
+  //     isLoading: true,
+  //   })
+
+  //   let data = new FormData()
+  //   data.append('file', event.target.files[0])
+
+  //   // Upload image
+  //   this.props.upload(data)
+  //     .then((response) => {
+  //       if (response.status === 200) {
+  //         this.props.messageShow('File uploaded successfully.')
+
+  //         let image = this.state.image
+  //         image = `/images/uploads/${response.data.file}`
+
+  //         this.setState({
+  //           image,
+  //         })
+  //       } else {
+  //         this.props.messageShow('Please try again.')
+  //       }
+  //     })
+  //     .catch((error) => {
+  //       this.props.messageShow('There was some error. Please try again.')
+  //     })
+  //     .then(() => {
+  //       this.setState({
+  //         isLoading: false,
+  //       })
+
+  //       window.setTimeout(() => {
+  //         this.props.messageHide()
+  //       }, 5000)
+  //     })
+  // }
+
+  render() {
+    return (
+      <div>
+        {/* SEO */}
+        <Helmet>
+          <title>Edit Profile - Crate</title>
+        </Helmet>
+
+        {/* Top Grey bar */}
+        <Grid style={{ backgroundColor: grey }} justifyRight={true}>
+          <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
+            <H3 font='secondary'>Edit profile</H3>
+          </GridCell>
+          
+          <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '9vw' }}>
+            <Button theme='secondary' onClick={this.props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+          </GridCell>
+        </Grid>
+        
+        <form style={{
```

Comment:
Great Job Team! Thanks for pulling the weight during my technical interview. You all are the best!